sanic/server.py

```diff
@@ -1,47 +1,47 @@
 import asyncio
+from functools import partial
 from inspect import isawaitable
 from signal import SIGINT, SIGTERM
 
-import httptools
+from httptools import HttpRequestParser
+from httptools.parser.errors import HttpParserError
 
 try:
     import uvloop as async_loop
 except ImportError:
     async_loop = asyncio
 
+from .config import Config
 from .log import log
 from .request import Request
 
+CONNECTIONS = set()
```

Comment:
It does imply a constant (it's not enforced though). It might be faster to have a dict of sets rather then member, objects are pretty slow in python. Once the low hanging stuff is cleaned up it will probably be worth it to pass over all the code and remove all the dots on the hot path.